internal/ls/folding_test.go

```diff
@@ -1101,12 +1099,6 @@ const[| {
 		const foo = null;
 
 		function Foo()[| {
-		  [|/**
-		    * Description
-		    *
-		    * @param {string} param
-		    * @returns
-		    */|]
```

Comment:
Does this crash, and that's why it's being removed? Can we leave the comment in and make sure we don't crash?

If we do crash, the editor will pop up an error message.

Reply:
Gotcha; that seems fine, though it seems like `visitNode` could just skip any JSDoc nodes too?

Reply:
Same reason for the other change in that same commit. 
Btw, why is it that we're testing `IsDeclarationNode(node)` and not `isDeclarationKind(node.Kind)`?

Reply:
I compared the debugging here with Strada, what's happening is that when the node is a `BinaryExpression`, then it returns true for `IsDeclarationNode(node)` which did not use to be the case in Strada for `isDeclarationKind(node.Kind)`. This is causing the duplication of that range.

Reply:
Oh, bleh, because BinaryExpressions are currently Declarations for JSDoc reasons. Can you easily special case that for now until that is resolved?